package.json

```diff
@@ -33,7 +33,7 @@
       "^.+\\.js$": "<rootDir>/node_modules/babel-jest",
       ".(ts|tsx)": "<rootDir>/node_modules/ts-jest/preprocessor.js"
     },
-    "testRegex": "(/__tests__/.*|\\.(test|spec))\\.(ts|tsx|js)$",
+    "testRegex": "/__tests__/.*-tests.(ts|tsx|js)$",
```

Comment:
I added some fixtured data in the tests, which caused it to be recognised by jest